cmd/execution_builder.go

```diff
@@ -801,12 +801,12 @@ func (exeNode *ExecutionNode) LoadRegisterStore(
 
 	if !bootstrapped {
 		checkpointFile := path.Join(exeNode.exeConf.triedir, modelbootstrap.FilenameWALRootCheckpoint)
-		root, err := exeNode.builder.RootSnapshot.Head()
+		sealedRoot, err := node.State.Params().SealedRoot()
```

Comment:
In dynamic bootstrapping, the sealed root is lower than finalized root (RootSnapshot.Head returns finalized root), EN needs to load [sealedRoot + 1, finalizedRoot] block range on startup.